lib/grover/js/processor.js

```diff
@@ -124,6 +124,18 @@ const _processPage = (async (convertAction, urlOrHtml, options) => {
       page.setGeolocation(geolocation);
     }
 
+    const failOnRequest = options.requestFailure; delete options.requestFailure;
+    if (failOnRequest !== undefined) {
+      page.on('requestfinished', (request) => {
+        if(request.response() && !request.response.ok()) {
```

Comment:
Prefer `if (`. Sorry, I haven't setup style linting for the JS code. Will need to add it to my list :)
```suggestion
        if (request.response() && !request.response.ok()) {
```